src/main/resources/swagger.api/publications.yaml

```diff
@@ -53,6 +53,19 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '500':
           $ref: '#/components/responses/InternalServerError'
+    delete:
+      summary: Delete publication result by Id
+      operationId: deletePublicationResultById
```

Comment:
Operation name `deletePublicationById` would be more relevant
summary value also needs to be updated accordingly

Reply:
Fixed.